src/components/counter.js

```diff
@@ -30,42 +35,80 @@ const styles = StyleSheet.create({
   }
 });
 
-class Learning_Project extends Component {
+export class Learning_Project extends Component {
   render() {
     return (
       <View
         style={{
           flex: 1,
-          padding: 20,
-          flexDirection: "row",
-          justifyContent: "center"
+          flexDirection: "column"
         }}
       >
-        <Button
-          large
-          title="Increment +"
-          color="green"
-          onPress={this.props.actions.counterIncrement}
-        />
-        <Text>
-          <Text style={styles.text}>{this.props.count}</Text>
-        </Text>
-        <Button
-          large
-          title="Decrement -"
-          color="red"
-          onPress={this.props.actions.counterDecrement}
-        />
+        <View
+          style={{
```

Comment:
Pull all of the styles out into their own variable and then use that variable to pass styles. See an example here: https://github.com/showclix/Live-React-Native/blob/dev/src/components/eventCell.tsx#L10. This helps keep a separation of styles, which you might have to use again, and JSX which makes the JSX much easier to read.